test/ctia/bulk/routes_test.clj

```diff
@@ -196,15 +196,16 @@
 (deftest test-bulk-wait_for-test
   ((:es-store store-fixtures)
```

Comment:
Idea: at some point a helper function that only tests for a subset of `store-fixtures` would be helpful.